jiant/preprocess.py

```diff
@@ -503,20 +544,20 @@ def update_vocab_freqs(sentence):
             for sentence in task.get_sentences():
                 update_vocab_freqs(sentence)
 
-    # This branch is meant for tasks that have *English* target sentences
-    # (or more generally, same language source and target sentences)
-    # Tasks with different language source and target sentences should
-    # count and return the vocab in a `task.all_labels()` method.
-    for task in tasks:
-        if hasattr(task, "target_sentences"):
-            for sentence in task.target_sentences:
-                update_target_vocab_freqs(sentence)
```

Comment:
This code block is being removed because `update_target_vocab_freqs` was removed in #306.